content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-static-web-app.md

```diff
@@ -85,22 +85,22 @@ jobs:
         with:
           submodules: true
       - name: Build And Deploy
-        uses: Azure/static-web-apps-deploy@1a947af9992250f3bc2e68ad0754c0b0c11566c9
+        uses: Azure/static-web-apps-deploy@v1
         with:
           azure_static_web_apps_api_token: {% raw %}${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}{% endraw %}
           repo_token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
           action: "upload"
           app_location: {% raw %}${{ env.APP_LOCATION }}{% endraw %}
           api_location: {% raw %}${{ env.API_LOCATION }}{% endraw %}
-          app_artifact_location: {% raw %}${{ env.APP_ARTIFACT_LOCATION }}{% endraw %}
+          output_location: {% raw %}${{ env.OUTPUT_LOCATION }}{% endraw %}
 
-  close:
+  close_pull_request:
     if: github.event_name == 'pull_request' && github.event.action == 'closed'
     runs-on: ubuntu-latest
-    name: Close
+    name: Close Pull Request
     steps:
-      - name: Close
-        uses: Azure/static-web-apps-deploy@1a947af9992250f3bc2e68ad0754c0b0c11566c9
+      - name: Close Pull Request
+        uses: Azure/static-web-apps-deploy@v1
```

Comment:
```suggestion
        uses: Azure/static-web-apps-deploy@1a947af9992250f3bc2e68ad0754c0b0c11566c9
```
Same as above, just switching this back to use the SHA value associated with v1.